assets/js/base/context/providers/cart-checkout/payment-methods/types.ts

```diff
@@ -27,17 +27,21 @@ export type CustomerPaymentMethods =
 	| Record< string, CustomerPaymentMethod >
 	| EmptyObjectType;
 
-export type PaymentMethodDispatchers = {
+export interface PaymentMethodDispatchers {
 	setRegisteredPaymentMethods: ( paymentMethods: PaymentMethods ) => void;
 	setRegisteredExpressPaymentMethods: (
 		paymentMethods: ExpressPaymentMethods
 	) => void;
 	setShouldSavePayment: ( shouldSave: boolean ) => void;
-};
+	setActivePaymentMethod: (
+		paymentMethod: string,
+		paymentMethodData?: ObjectType | EmptyObjectType
```

Comment:
Data can now be set at the same time as setting an active payment method. This removes the need to use STARTED status to set data.